### ISSUE TITLE
Closing a tab with open VR pane causes errors on Win10

### ISSUE THREAD

**Issue by gatesphere** [leo-editor/leo-editor#4009]
See PR #4014.

My env:

```
Leo 6.8.1-devel, devel branch, build e21cd1ebe4
2024-07-16 16:46:21 -0500
Python 3.10.6, PyQt version 6.6.3
Windows 10 AMD64 (build 10.0.19045) SP0
```

Repro steps:

- Open new .leo file (doesn't need to be saved) in a new tab
- Type some stuff into the body
- Alt-0 to open vr pane
- Close the new .leo file tab while vr pane is visible, don't save
- Get repeating errors in log and console:

```
Traceback (most recent call last):

  File "<redacted>\repos\leo-editor\leo\plugins\viewrendered.py", line 805, in update
    sb = w.verticalScrollBar()

RuntimeError: wrapped C/C++ object of type QTextBrowser has been deleted
```

(This repeats several times, about once a second, and continues until Leo is closed entirely.)

**Comment by gatesphere** [comment]
Leo is definitely still usable in this instance, so I hesitate to add the 'Critical' tag, but this is not a *fun* experience.

Cleanup of the VR pane object appears to be missing when a tab is closed.

**Comment by gatesphere** [comment]
Confirmed this behavior also happens on same commit on my Arch Linux environment, so this is not a Windows-specific bug.

**Comment by tbpassin** [comment]
I just repo'ed the bug in Win10.  Interestingly it didn't happen with VR3. Maybe that's because VR3 doesn't know how to set the vertical scrollbar yet...

There's probably a `deleteLater()` that needs to happen somewhere.  The bug may have surfaced with a particular version of Qt.  It could have been happening all along but not getting announced, perhaps.

**Comment by edreamleo** [comment]
Closed via PR #4014.
